doc/nrf/ug_edge_impulse.rst

```diff
@@ -109,11 +110,33 @@ Complete the following steps to do this:
 
       Project ID in Edge Impulse studio dashboard
 
-#. Define the :c:macro:`EI_API_KEY_HEADER` CMake option as ``x-api-key:aaaabbbbccccdddd`` to provide the x-api-key associated with yout Edge Impulse project.
+#. Define the :c:macro:`EI_API_KEY_HEADER` CMake option (see :ref:`cmake_options`) as ``x-api-key:[ei api key]`` to provide the x-api-key associated with yout Edge Impulse project.
    You can access your API keys under the :guilabel:`Keys` tab in the Edge Impulse project dashboard.
 
    .. figure:: images/ei_api_key.png
       :scale: 50 %
       :alt: API key under the Keys tab in Edge Impulse studio
 
       API key under the Keys tab in Edge Impulse studio
+
+Complete the following steps to download model from public Edge Impulse project:
+
+1. Check an ID of the public project.
+   This ID is different from an ID shown in the Edge Impulse Studio in the :guilabel:`Project info` panel.
+   You can do this by visiting this address:
+   .. parsed-literal::
+      :class: highlight
+
+      https:\ //studio.edgeimpulse.com/v1/api/*XXXXX*/versions/public
+
+   The *XXXXX* must be set to the ID shown in the Edge Impulse Studio in the :guilabel:`Project info` panel.
+   The ID of the public project is displayed next to the "publicProjectId" field.
+
+#. Set :kconfig:`CONFIG_EDGE_IMPULSE_URI` to the following URI from Edge Impulse studio:
+
+   .. parsed-literal::
+      :class: highlight
+
+      CONFIG_EDGE_IMPULSE_URI="https:\ //studio.edgeimpulse.com/v1/api/*XXXXX*/deployment/download?type=zip&modelType=int8"
```

Comment:
I would add a note explaining why modelType=int8 is added here (that we can only download a quantized EON compiled model).